optax/_src/base_test.py

```diff
@@ -15,8 +15,11 @@
 """Tests for base.py."""
 
 from absl.testing import absltest
+
 import chex
 import numpy as np
```

Comment:
(Pylint will raise a wrong import order error here)

Reply:
I'll fix this change, but running `test.sh` locally did not raise this issue for me (tried running `pylint --rcfile=.pylintrc optax/_src/base_test.py` separately as well).

Reply:
Thanks for letting me know! This would definitely be blocked internally as we try to merge so I'll look into adding an import order check to `test.sh`.